dependencies.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+ * Build dependencies
+ */
+ext.androidCompileSdkVersion = 26
+ext.androidMinSdkVersion = 19
```

Comment:
There's no reason to bump this to 19. Knock it back down to 14.

Reply:
Wait, did you even use this anywhere? 😆 
Copy/paste error?

Reply:
This was totally a copy/paste error, some things never change 😂